app/src/test/java/org/mozilla/fenix/home/RecentTabsListFeatureTest.kt

```diff
@@ -137,8 +139,10 @@ class RecentTabsListFeatureTest {
         homeStore.waitUntilIdle()
 
         assertEquals(2, homeStore.state.recentTabs.size)
-        assertEquals(selectedTab, homeStore.state.recentTabs[0])
-        assertEquals(mediaTab, homeStore.state.recentTabs[1])
+        assert(homeStore.state.recentTabs[0] is RecentTab.Tab)
```

Comment:
nit: `assertTrue` here and all the other places below for this PR.

```suggestion
        assertTrue(homeStore.state.recentTabs[0] is RecentTab.Tab)
```